src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -296,6 +296,10 @@ public static partial class RoslynInsertionTool
                                 Console.WriteLine("Could not create diff links.");
                             }
 
+                            Console.WriteLine("---PR description---");
+                            Console.WriteLine(prDescription);
```

Comment:
Eventually I'd like to have a dry-run flag for local testing so we can have decent experience finally getting to the bottom of #502